### ISSUE TITLE
Autoplay not disabled on this page...

### ISSUE THREAD

**Issue by martinfabian** [Eloston/disable-html5-autoplay#59]
I don't know if we are supposed to post specific things like this as issues. If not, just remove this one.

On this page http://www.dn.se/nyheter/sverige/forsvunnen-76-aring-tillratta/ there is a video ad that autoplays even though the app claims to disable autoplay. And it autoplays on each re-load of the page as well. This is Chrome Version 51.0.2704.106 m on W10x64, Surface Pro 3, with Adobe Flash plugin disabled.

Thanks.

**Comment by Eloston** [comment]
Yeah it's fine if you post specific things like this.

Unfortunately, I can't easily reproduce your issue. The advertisements that show depend on a lot of factors that are not easy to control, so I'm not always able to get HTML5 media advertisements or even the same advertisements every time. If you could somehow provide a link or some steps to get the advertisement, then that could work.

**Comment by martinfabian** [comment]
OK thanks, I also get different ads when visiting that page on different occasions. I have no clue how to consistently get videos. However, it seems that on most of the pages under that same domain video ads autoplay, like also here http://www.dn.se/nyheter/varlden/massiv-ilska-i-skottland/ 

If I right-click the video and open it in another tab https://r4---sn-5go7yn7z.c.2mdn.net/videoplayback/id/050dd38576ceb37b/itag/46/source/doubleclick_dmm/ratebypass/yes/acao/yes/ip/155.4.128.223/ipbits/0/expire/3608797929/sparams/acao,expire,id,ip,ipbits,itag,mm,mn,ms,mv,nh,pl,ratebypass,source/signature/3FE8B0E85AE0930C3D7DB628314DD602B0C67F9F.5E2817624119A5EF10C487FFF275531B7DFEF1A6/key/cms1/cms_redirect/yes/mm/30/mn/sn-5go7yn7z/ms/nxu/mt/1466925286/mv/m/nh/IgpwcjAxLmFybjAxKgkxMjcuMC4wLjE/pl/18/file/file.webm
it does not autoplay, and it seems that it actually gets blocked by the app. But apparently it somehow flies under the radar when embedded on the original page.

I looked into the source but I do not understand much of the javascript that runs those pages. Sorry.

**Comment by Eloston** [comment]
The video you linked is a webm file (note the file extension at the end), so Chrome automatically generates a page to make that vieweable.

Upon closer inspection in your second URL, it seems that your IP address might have been in there, so I edited your post and redacted that link for you. It seems that asking for the video URL wasn't such a good idea...

Although I was unlucky and couldn't get any videos to show up, I noticed that their advertisements ran in blank iframes, so this is probably related to #24. Once that gets fixed, we can re-test and see if anything changes.

**Comment by CA-girl** [comment]
This page, http://www.pcworld.com/article/3119886/software/stop-html5-autoplay-videos-in-chrome-firefox-opera-but-not-edge-explorer-safari.html, which ironically tells you how to stop autoplay videos in Chrome STARTS with an autoplay video. Argh!  It suggests that I add your extension to Chrome, which I've already done & enabled.  So, evidently it can't stop videos playing on the PCWorld website.

**Comment by yegeniy** [comment]
@CA-girl it does stop the ads autoplaying for me, as of today; however, I am unable to manually play the ad.


* The URL to X (or steps to get to the page that breaks)
http://www.pcworld.com/article/3119886/software/stop-html5-autoplay-videos-in-chrome-firefox-opera-but-not-edge-explorer-safari.html
* What media on the page breaks (if there is multiple)
`div.vjs-ima3-ad-container`
* Steps that cause it to break
Trying to press the Pause / Play button manually causes an "Unknown Source" autoplay attempt to be registered in the extension's autoplay counter.
* The extension version, browser version, and OS type
`Disable HTML5 Autoplay 0.6.2`, `Chrome Version 56.0.2924.87 (64-bit)`, `Mac OS X Sierra 10.12.1`
* Anything else that may be related to this issue
I want to play my ad manually, I just don't want it to autoplay before I'm done reading the content of the page.
